bears/xml/XMLBear.py

```diff
@@ -22,12 +22,13 @@ class XMLBear(LocalBear, Lint):
     diff_message = "XML can be formatted better."
     output_regex = r'(.*\.xml):(?P<line>\d+): (?P<message>.*)\n.*\n.*'
     gives_corrected = True
+    use_stderr = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
             return self._process_issues(self.stderr_output, filename)
+        if self.stdout_output:
+            return self._process_corrected(self.stdout_output, filename, file)
```

Comment:
@sils1297 I switched it because if there is an error in stderr then return no need to check stdout else check if something needs to be corrected